statsmodels/compat/pandas.py

```diff
@@ -8,14 +8,18 @@
 version = LooseVersion(pandas.__version__)
 pandas_lte_0_19_2 = version <= LooseVersion('0.19.2')
 pandas_gt_0_19_2 = version > LooseVersion('0.19.2')
-
+pandas_ge_20_0 = version >= LooseVersion('0.20.0')
+pandas_ge_25_0 = version >= LooseVersion('0.25.0')
 
 try:
     from pandas.api.types import is_numeric_dtype  # noqa:F401
 except ImportError:
     from pandas.core.common import is_numeric_dtype  # noqa:F401
 
-if version >= '0.20':
+if pandas_ge_25_0:
+    from pandas.tseries import frequencies  # noqa:
```

Comment:
what about frequencies?

Reply:
fixed typo for the end of this line.  Were you referring to something else?

How much of this do we actually use?  e.g. pandas.util.testing we frequently import directly in the tests.

Reply:
I missed frequencies since the try/ex wasn't needed.   LGTM.

Reply:
Great.  If we can get this into a 0.10.1 we can prevent a months-long repeat of the scipy import errors.